tests/inspect/inspect.go

```diff
@@ -58,6 +62,26 @@ func main() {
 		fmt.Fprintf(os.Stdout, "%s=%s\n", globalFlags.PrintEnv, os.Getenv(globalFlags.PrintEnv))
 	}
 
+	if globalFlags.PrintCapsPid >= 0 {
+		const linuxCapabilityVersion3 = 0x20080522 // as documented in capget(2)
+		hdrp := []uint32{linuxCapabilityVersion3, uint32(globalFlags.PrintCapsPid)}
+		datap := []uint32{0, 0, 0, 0, 0, 0, 0}
+		_, _, errno := syscall.Syscall(syscall.SYS_CAPGET, uintptr(unsafe.Pointer(&hdrp[0])), uintptr(unsafe.Pointer(&datap[0])), 0)
```

Comment:
Hm, reading capHeader and capget (https://github.com/syndtr/gocapability/blob/master/capability/syscall_linux.go#L14) makes me wonder what you did wrong, when you tried to use structs. Yours looked the same as those.